src/main/java/org/dimdev/riftloader/RiftLoader.java

```diff
@@ -29,7 +29,8 @@
     public static final RiftLoader instance = new RiftLoader();
     private static final Logger log = LogManager.getLogger("RiftLoader");
 
-    public final File modsDir = new File(Launch.minecraftHome, "mods");
+    public File modsDir = new File(Launch.minecraftHome, "mods");
+    public final File modsVersionSpecifiedDir = new File(Launch.minecraftHome, "mods/1.13-Rift");
```

Comment:
Since we are talking about making Rift mods at least cross 1.13 compatible, I'd suggest just making the folder Rift and making an exception for if the folder exists, search there. And please don't have the folder just created, just check to see if it's existing already.

Reply:
OK let me think a little bit....
I agree that cross compatibility is a very good idea. So maybe create a folder called `RiftMods` under `.minecraft/`?

Here is the rough draft. For backward compatibility, Rift's mod will load by this order:
1.  Create the directory called `RiftMods`, so ppl know where to put their mods
2. Try to see if the folder `RiftMods/_version_` exists, if there are any mods inside, load them
3. If not, try to see if there are any mods in `RiftMods/`, load from there
4. If not, load mods from `mods/` just like the old days

What do you think?

Reply:
What? No, the way it works is fine, just modify what you have from `1.13-Rift` to just `Rift` and just rather than straight up creating the folder, just add an if statement in the search to see if it exists, and if it does, search it.